### PR TITLE
Business Docs -> Staging

### PR DESCRIPTION
**Pull request by Chinoman10**

### CODE REVIEW

File: docs/public-api/notification/wallet-monitor.mdx [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+          "status": "CREATED",
+          "timestamp": 1682174193510
+      },
+      "status": 200
+  }
+```  
+
+:::info
+
+In this section we only show the notification parameters  specific for liquid staking. To see the rest of the body structure please check the [notification](/public-api/notification)
```
Comment (NunoD87) [inline docs/public-api/notification/wallet-monitor.mdx]: Change `liquid staking` to `wallet monitor`

File: docs/public-api/notification/liquid-staking.mdx [hunk 2]
```diff
@@ -1,43 +1,45 @@
...
- | notificationParams.finishUnbond.value   | optional    | Float   | value from when the finishUnbond is taken in consideration                                          |
- | notificationParams.finishUnbond.type    | optional    | String  | use [priceType](/public-api/ids) varible value from when the finishUnbond is taken in consideration |
- | notificationParams.scope                | mandatory   | String  | use [scope](/public-api/ids) varible value                                                          |
+ | Parameter            | Requirement | Type    | Description                                                                                             |
+ | :------------------- | :---------- | :------ | :------------------------------------------------------------------------------------------------------ | 
+ | contract             | mandatory   | String  | Params of the notification                                                                              |
+ | tokenContract        | mandatory   | String  | Params of the notification                                                                              |
```
Comment (NunoD87) [inline docs/public-api/notification/liquid-staking.mdx]: Maybe change on contract and tokenContract the description `Params of the notification` to `Smart Contract Address` and `Bonded Token Address`, respectively.

File: docs/intro.mdx [hunk 3]
```diff
@@ -29,10 +29,22 @@ With the **Governance Poll** notification, subscribe to governance polls created
...
+### 👛 Wallet Watcher
 
 The **Wallet Watcher** notification allow users to follow all transactions made to or from one or multiple wallets.
 
+### 💼 Validator Profile changes
+
+### 💼 Liquid Staking
+
+### 💼 Liquidity Pool
+
```
Comment (NunoD87) [inline docs/intro.mdx:45]: Perhaps remove those that have no descriptions.

File: docs/webapp/sign-up.mdx [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+![Sign up screen](/img/signup/mainscreen.png)
+
+2- Click the Connect Wallet button
+
+3- Read and accept the [Terms of Use](https://hermesprotocol.io/terms) and [Privacy Policy](https://hermesprotocol.io/privacy_policy) by checking the box ✅
+
+4- After confirming you can choose the wallet that you wish to use to connect:
+![choose wallet pop-up](/img/signup/choose_wallet.png)
+
+5- If you do not have one of the wallet extensions, they'll prompt you to Install them. Pick your favorite and proceed. Note: if you just installed an extension, make sure to create or import a wallet before trying to proceed.
```
Comment (NunoD87) [inline docs/webapp/sign-up.mdx]: Change the `Note: ...` to a Docusaurus' Admonitions

```mdx
:::note

If you just installed an extension, make sure to create or import a wallet before trying to proceed.

:::
```

File: docs/business-webapp/sign-up.mdx [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+![Sign up screen](/img/signup/mainscreen.png)
+
+2- Click the Connect Wallet button
+
+3- Read and accept the [Terms of Use](https://hermesprotocol.io/terms) and [Privacy Policy](https://hermesprotocol.io/privacy_policy) by checking the box ✅
+
+4- After confirming you can choose the wallet that you wish to use to connect:
+![choose wallet pop-up](/img/signup/choose_wallet.png)
+
+5- If you do not have one of the wallet extensions, they'll prompt you to Install them. Pick your favorite and proceed. Note: if you just installed an extension, make sure to create or import a wallet before trying to proceed.
```
Comment (NunoD87) [inline docs/business-webapp/sign-up.mdx]: Change the `Note: ...` to a Docusaurus' Admonitions

```mdx
:::note

If you just installed an extension, make sure to create or import a wallet before trying to proceed.

:::
```

File: docs/business-webapp/broadcast/get_subscribers/telegram.mdx [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+
+![create list](/img/broadcast/telegram-auth.png)
+
+:::info
+You can only invite the bot to a Group where you are an Admin.
+:::
+
+Once the bot is successfully added to your community, use the following command to generate the subscribe List embed:
+
+... (work in progress) ...
```
Comment (NunoD87) [inline docs/business-webapp/broadcast/get_subscribers/telegram.mdx]: Perhaps set this page as a draft to not be built, as it's a work in progress yet.

```mdx
---
title: Get subscribers on Telegram
sidebar_position: 2
sidebar_label: Telegram
draft: true
---
```

Comment (Duarib) [reply docs/business-webapp/broadcast/get_subscribers/telegram.mdx]: Left it without so it doesn't break links

**Pull request merged**